docs/sphinx/formats/dataset-table.txt

```diff
@@ -398,12 +398,12 @@ to open/import a dataset in a particular format.
    * - SM Camera
      - (no extension)
      - Single file
+   * - SPC FIFO Data
```

Comment:
Ah, I see, it isn't that I forgot it, it's that I updated the reader but didn't regen the dataset structure table.